index.js

```diff
@@ -83,23 +83,36 @@ initIstanbulHookHack = function (options) {
     Module._extensions['.js'] = function (module, filename) {
         var src = fs.readFileSync(filename, {encoding: 'utf8'}),
             tmp;
+            
```

Comment:
@zordius This is the file you want to look at. It can be hard to find the right one amongst all the deleted files from the 'coverage' directory that were deleted.